pilot/pkg/networking/core/v1alpha3/loadbalancer/loadbalancer.go

```diff
@@ -245,6 +250,22 @@ func applyPriorityFailover(
 	loadAssignment.Endpoints = localityLbEndpoints
 }
 
+// Returning the label names in a separate array as the iteration of map is not ordered.
+func priorityLabelOverrides(labels []string) ([]string, map[string]string) {
+	priorityLabels := make([]string, 0, len(labels))
+	overriddenValueByLabel := make(map[string]string, len(labels))
+	var tempStrings []string
+	for _, labelWithValue := range labels {
+		tempStrings = strings.Split(labelWithValue, string(FailoverPriorityLabelDefaultSeparator))
+		priorityLabels = append(priorityLabels, tempStrings[0])
+		if len(tempStrings) == 2 {
+			overriddenValueByLabel[tempStrings[0]] = tempStrings[1]
```

Comment:
It seems you are allowing misuse `label` and `label=value`, looks a little hard to understand

Reply:
make some sense to me.